### PR TITLE
74-add parquet for temp data and avoid explicit column conversions 

### PR DESCRIPTION
**Pull request by lboel**
#74

### CODE REVIEW

File: R/helper_main.R [hunk 1]
```diff
@@ -134,22 +134,15 @@ read_column_synonyms <- function(synonym_file) {
...
-                        suffix,
-                        ".csv"
-                    )
-                ),
-            row.names = F,
-            na = "",
-            fileEncoding = "UTF-16LE",
-            quote = T
-        )
+    data %>% arrow::write_parquet( sink =
```
Comment (pmayd) [inline R/helper_main.R:137]: does arrow also omit the row.names? What about missing values? So compared to `write.csv` what about the parameters `row.names`, `na`?

**Review by pmayd — APPROVED** [review]
Nice and thanks that you took care of this! 

However, I think we also have to adjust our final pipeline if you change the central `export_data` function to parquet as our upload uses the final CSV files in the table folder. 

But I don't think you have to handle this, I can take over and see how bq can work with parquet files!

**Comment by pmayd** [comment]
It is possible and we no longer need a schema file along with the upload! https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-parquet

**Comment by lboel** [comment]
please note:

 I plan to tidy up everything tonight. But the output works so far.

So after fixing some issues now with the logging system, i see a quick win here.

Were noticing a performance improvement after removing the conversions as well, so script executing should at least feel faster after we have finished this.

**Comment by lboel** [comment]
> It is possible and we no longer need a schema file along with the upload! https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-parquet
> 
> As you can see the file is just presented and bq is taking the schema directly from the parquet file, exactly what I wanted

Sorry my github app wasnt showing your comments in the afternoon.

Indeed, I have used parquet for uploading static clinic data to avoid defining the schema by hand.

Works perfectly. 

For the temp files, we can even use a memory based file write logic with a single parquet file holding a tibble of filename and content at the end of script 1 and 2.

Will work on a mvp now and tomorrow morning. 

 eventually, reducing the total of file writes to the file system afterwards.

**Comment by pmayd** [comment]
I am not sure I would want to have the intermediated files not saved to the disk...if everything works fine, but maybe we realize somewhat later that there is a problem with the pipeline and that there are strange things happening in script 2, I had this now several times. And it is really easy to debug if you can check whether the output of script 1 already has the same bug or error or if this is introduced by the second script. So for now I would leave this as is

**Comment by lboel** [comment]
Close due to too many merge conflicts because to many new lines in develop in favor for a new clean branch - here: #98

**Pull request closed without merging**